catalog/core/catalog-core-camelcomponent/src/main/java/ddf/camel/component/catalog/content/ContentProducerDataAccessObject.java

```diff
@@ -192,25 +189,15 @@ public void createContentItem(FileSystemPersistenceProvider fileIdMap, ContentEn
 
     public void processHeaders(Map<String, Object> headers, StorageRequest storageRequest,
             File ingestedFile) {
-        String attributeOverrideHeaders = (String) headers.get(Constants.ATTRIBUTE_OVERRIDES_KEY);
-        if (StringUtils.isNotEmpty(attributeOverrideHeaders)) {
+        HashMap<String, Serializable> attributeOverrideHeaders = new HashMap<>((Map) headers.get(
```

Comment:
As for the NPE, it looks like it'll either be a populated or an empty list. If nothing is given from the front it end it gives an empty list and it won't be used anywhere since they're all checking if it's empty or not :+1: